docs/src/tools.md

```diff
@@ -39,3 +39,68 @@ cargo install mdbook
+
+1. **PYBUDA_EXTRACT_UNIQUE_OP_CONFIG_AT**
+   - By setting this flag to one of the following options, the model's unique ops configuration can be extracted at a specific compilation stage or across all stages:
+
+     - **`PYBUDA_EXTRACT_UNIQUE_OP_CONFIG_AT = ALL`**
+       Extracts all the unique ops configurations present in the graph at every compilation stage.
+
+     - **`PYBUDA_EXTRACT_UNIQUE_OP_CONFIG_AT = {INITIAL_GRAPH / PRE_OPTIMIZE_GRAPH / POST_OPTIMIZE_GRAPH / POST_AUTOGRAD_GRAPH / PRE_LOWERING_GRAPH}`**
+       Extracts the unique ops configuration only at the specified compilation stage.
+
+2. **PYBUDA_PRINT_UNIQUE_OP_CONFIG**
+   - By setting this flag to `1`, all unique configurations will be printed to the console.
+
+3. **PYBUDA_EXPORT_UNIQUE_OP_CONFIG_TO_CSV**
+   - By setting this flag to `1`, all unique configurations will be exported to a CSV file. The file can be saved to the default path (i.e., the current directory), or it can be saved to a specific path by setting the `PYBUDA_EXPORT_UNIQUE_OP_CONFIG_DIR_PATH` environment variable.
+
+> **Note:**
+> The delimiter used in the CSV file will be a hyphen (`-`) to avoid potential parsing issues. Commas (`,`) may appear in the op shapes and attributes, which could lead to misinterpretation of the data.
+
+## Cross Correlate Models and Ops
+
+The models and ops can be cross-correlated by running the `scripts/export_models_ops_correlation.py` python script.
+
+The script will perform the following tasks:
+
+1. Run all models until the compile depth specified by the user.
+2. Export unique op requirements to a file (each model variants has its own directory, in that directory each compile depth has its own file).
+3. Parse those unique op requirements and create a xlsx file that can be loaded into a google sheet.
+   1. The xlsx file will contain list of models on X axis (i.e. columns) and list of ops on Y axis (i.e. rows/indices).
+   2. Elements in between will contain a checkmark if the desired op from the Y axis (i.e., rows/indices) exists in the model on X axis (i.e., columns).
+   3. Models will be sorted alphabetically.
+   4. Ops will be sorted by the number of occurrences in the models.
+
+### Usage
+
+To run the script, use the following command:
+
+```sh
+python scripts/export_models_ops_correlation.py
+```
+
+### Required Options:
+
+|                              **Option**                            |                         **Description**                                                                          |
+| :----------------------------------------------------------------: | :--------------------------------------------------------------------------------------------------------------: |
+| `-c`, `--compile_depth {GENERATE_INITIAL_GRAPH,PRE_LOWERING_PASS}` | Choose the compilation depth for extracting ops configuration for the models present in `pytest_directory_path`. |
```

Comment:
Can we modify it like this to clarify to the user that there are more options:
```py
GENERATE_INITIAL_GRAPH, PRE_LOWERING_PASS, etc.
```

Reply:
Updated !